pixi_docs/src/main.rs

```diff
@@ -0,0 +1,273 @@
+use clap::builder::Str;
+use clap::{Command, CommandFactory, Parser};
+use itertools::Itertools;
+use std::error::Error;
+use std::fmt::Write;
+use std::fs;
+use std::path::{Path, PathBuf};
+
+const MD_EXTENSION: &str = ".md";
+// const RECURSIVE_COMMANDS: &[&str] = &["pixi", "project", "global"];
+
+#[derive(Parser)]
+#[command(about = "Generates markdown documentation for the Pixi CLI")]
+struct Cli {
+    /// Output directory for the generated documentation
+    #[arg(short, long, default_value = "../docs/autogenerated")]
+    output_dir: PathBuf,
+}
```

Comment:
i think this doesn't need to be configurable, let's just hardcode the output path relative to the Cargo manifest dir